src/EventStore.Rags.Tests/EventStore.Rags.Tests.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net46</TargetFramework>
+    <Platform>x64</Platform>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <OutputPath>..\..\bin\tests\</OutputPath>
```

Comment:
that make a lot harder to run `dotnt test` with multitargeting (useful later to test both `net461` and `netcoreapp`)

using `dontet test` is easier also from command line and build scripts, no need to use custom test runner

Reply:
This is mostly legacy from the current build scripts, and will be changed as per your comment.